ts/Stock/Indicators/DEMA/DEMAIndicator.ts

```diff
@@ -0,0 +1,256 @@
+/* *
+ *
+ *  License: www.highcharts.com/license
+ *
+ *  !!!!!!! SOURCE GETS TRANSPILED BY TYPESCRIPT. EDIT TS FILE ONLY. !!!!!!!
+ *
+ * */
+
+'use strict';
+
+import type {
+    DEMAOptions,
+    DEMAParamsOptions
+} from './DEMAOptions';
+import type DEMAPoint from './DEMAPoint';
+import type IndicatorValuesObject from '../IndicatorValuesObject';
+import type LineSeries from '../../../Series/Line/LineSeries';
+import BaseSeries from '../../../Core/Series/Series.js';
+const {
+    seriesTypes: {
+        ema: EMAIndicator
+    }
+} = BaseSeries;
+import RequiredIndicatorMixin from '../../../Mixins/IndicatorRequired.js';
+import U from '../../../Core/Utilities.js';
+const {
+    correctFloat,
+    isArray,
+    merge
+} = U;
+
+
+/**
+ * Internal types
+ * @private
+ */
+declare global {
+    namespace Highcharts {
+        interface DEMAIndicatorLinkedParentSeries extends LineSeries {
+            EMApercent?: number;
+        }
+    }
+}
+
+/**
+ * The DEMA series Type
+ *
+ * @private
+ * @class
+ * @name Highcharts.seriesTypes.dema
+ *
+ * @augments Highcharts.Series
+ */
+class DEMAIndicator extends EMAIndicator {
+    /**
+     * Double exponential moving average (DEMA) indicator. This series requires
+     * `linkedTo` option to be set and should be loaded after the
+     * `stock/indicators/indicators.js` and `stock/indicators/ema.js`.
+     *
+     * @sample {highstock} stock/indicators/dema
+     *         DEMA indicator
+     *
+     * @extends      plotOptions.ema
+     * @since        7.0.0
+     * @product      highstock
+     * @excluding    allAreas, colorAxis, compare, compareBase, joinBy, keys,
+     *               navigatorOptions, pointInterval, pointIntervalUnit,
+     *               pointPlacement, pointRange, pointStart, showInNavigator,
+     *               stacking
+     * @requires     stock/indicators/indicators
+     * @requires     stock/indicators/ema
+     * @requires     stock/indicators/dema
+     * @optionparent plotOptions.dema
+     */
+    public static defaultOptions: DEMAOptions = merge(EMAIndicator.defaultOptions, {
+
+    } as DEMAOptions)
```

Comment:
If there were not default options, you can just assign `merge(EMAIndicator.defaultOptions);`